pkgs/servers/x11/xorg/default.nix

```diff
@@ -2744,11 +2744,11 @@ lib.makeScope newScope (self: with self; {
   }) {};
 
   xrandr = callPackage ({ stdenv, pkgconfig, fetchurl, libX11, xorgproto, libXrandr, libXrender }: stdenv.mkDerivation {
-    name = "xrandr-1.5.0";
+    name = "xrandr-1.5.1";
     builder = ./builder.sh;
     src = fetchurl {
-      url = "mirror://xorg/individual/app/xrandr-1.5.0.tar.bz2";
-      sha256 = "1kaih7rmzxr1vp5a5zzjhm5x7dn9mckya088sqqw026pskhx9ky1";
+      url = "mirror://xorg/individual/app/xrandr-1.5.1.tar.xz";
```

Comment:
```suggestion
url = "mirror://xorg/individual/app/${name}}.tar.xz";
```

Reply:
>\# THIS IS A GENERATED FILE.  DO NOT EDIT!

Can be changed here, but not relevant to this pull request: https://github.com/NixOS/nixpkgs/blob/90bc70bbd2a1164f1dd98946f6732ac971fca102/pkgs/servers/x11/xorg/generate-expr-from-tarballs.pl#L308